Track.releases must not be null in bootstrapReleasePlayResources

### Describe the bug
```
bootstrapRegularAlphaPlayResources FAILED
 FAILURE: Build failed with an exception.
What went wrong:
 Execution failed for task 'bootstrapRegularAlphaPlayResources'
> track.releases must not be null


Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task 'bootstrapRegularAlphaPlayResources'.


org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)

Caused by: java.lang.IllegalStateException: track.releases must not be null

1com.github.triplet.gradle.play.tasks.Bootstrap.bootstrapReleaseNotes(Bootstrap.kt:111)
com.github.triplet.gradle.play.tasks.Bootstrap.access$bootstrapReleaseNotes(Bootstrap.kt:24)
com.github.triplet.gradle.play.tasks.Bootstrap$bootstrap$1.invoke(Bootstrap.kt:43)
com.github.triplet.gradle.play.tasks.Bootstrap$bootstrap$1.invoke(Bootstrap.kt:24)
com.github.triplet.gradle.play.internal.PlayPublishTaskBase.read(PlayPublishTaskBase.kt:74)
com.github.triplet.gradle.play.tasks.Bootstrap.bootstrap(Bootstrap.kt:38)
org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)

```

A clear and concise description of what the bug is.
I want to publish an APK to alpha channel. I've specified the track as 'alpha' but everytime the build fails with the error 'track.releases must not be null'

### To Reproduce

Steps to reproduce the behavior (what tasks are you running, what's your `play` config, etc.)

--- a/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/Bootstrap.kt
+++ b/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/Bootstrap.kt
@@ -108,7 +108,7 @@ open class Bootstrap : PlayPublishTaskBase() {
     private fun AndroidPublisher.Edits.bootstrapReleaseNotes(editId: String) {
         progressLogger.progress("Downloading release notes")
         tracks().list(variant.applicationId, editId).execute().tracks?.forEach { track ->
-            track.releases.maxBy {
+            track.releases?.maxBy {
                 it.versionCodes?.max() ?: Long.MIN_VALUE
             }?.releaseNotes?.forEach {
                 File(srcDir, "$RELEASE_NOTES_PATH/${it.language}/${track.track}.txt")

